src/app/public/modules/country-field/fixtures/country-field.component.fixture.html

```diff
@@ -0,0 +1,12 @@
+<div>
+  <form>
+    <sky-country-field
+      [(ngModel)]="modelValue"
```

Comment:
Two way bound properties should come last.

Reply:
Fixed